src/main/java/org/antipathy/mvn_scalafmt/FormatMojo.java

```diff
@@ -25,18 +28,22 @@
     private boolean skipSources;
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     private MavenProject project;
+    @Parameter(defaultValue = "${project.build.sourceDirectory}", required = true)
+    private List<File> sourceDirectories;
+    @Parameter(defaultValue = "${project.build.testOutputDirectory}", required = true)
```

Comment:
Would this be better moving down into logic in the class?  I notice that when `testSourceDirectories` is not specified that the formatter attempts to format `target/test-classes`.  Seems to be resolved by changing to:

```
if (!skipSources) {
            if (sourceDirectories == null) {
                sources.addAll(project.getCompileSourceRoots());
            } else {
                sources.addAll(sourceDirectories);
            }
        }

        if (!skipTestSources) {
            if (testSourceDirectories == null) {
                testSources.addAll(project.getTestCompileSourceRoots());
            } else {
                testSources.addAll(testSourceDirectories);
            }
        }
```